services/lambda/src/it/java/software/amazon/awssdk/services/lambda/ServiceIntegrationTest.java

```diff
@@ -1,221 +0,0 @@
-/*
- * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License").
- * You may not use this file except in compliance with the License.
- * A copy of the License is located at
- *
- *  http://aws.amazon.com/apache2.0
- *
- * or in the "license" file accompanying this file. This file is distributed
- * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- * express or implied. See the License for the specific language governing
- * permissions and limitations under the License.
- */
-
-package software.amazon.awssdk.services.lambda;
-
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.nio.charset.StandardCharsets;
-import java.util.concurrent.TimeUnit;
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Rule;
-import org.junit.Test;
-import software.amazon.awssdk.core.SdkBytes;
-import software.amazon.awssdk.services.lambda.model.CreateEventSourceMappingRequest;
-import software.amazon.awssdk.services.lambda.model.CreateEventSourceMappingResponse;
-import software.amazon.awssdk.services.lambda.model.CreateFunctionResponse;
-import software.amazon.awssdk.services.lambda.model.DeleteEventSourceMappingRequest;
-import software.amazon.awssdk.services.lambda.model.DeleteFunctionRequest;
-import software.amazon.awssdk.services.lambda.model.FunctionCode;
-import software.amazon.awssdk.services.lambda.model.FunctionConfiguration;
-import software.amazon.awssdk.services.lambda.model.GetEventSourceMappingRequest;
-import software.amazon.awssdk.services.lambda.model.GetEventSourceMappingResponse;
-import software.amazon.awssdk.services.lambda.model.GetFunctionConfigurationResponse;
-import software.amazon.awssdk.services.lambda.model.GetFunctionResponse;
-import software.amazon.awssdk.services.lambda.model.InvocationType;
-import software.amazon.awssdk.services.lambda.model.InvokeRequest;
-import software.amazon.awssdk.services.lambda.model.InvokeResponse;
-import software.amazon.awssdk.services.lambda.model.ListFunctionsRequest;
-import software.amazon.awssdk.services.lambda.model.ListFunctionsResponse;
-import software.amazon.awssdk.services.lambda.model.LogType;
-import software.amazon.awssdk.services.lambda.model.Runtime;
-import software.amazon.awssdk.testutils.retry.RetryRule;
-import software.amazon.awssdk.utils.BinaryUtils;
-
-public class ServiceIntegrationTest extends IntegrationTestBase {
```

Comment:
This might be a good one to keep for sample code purpose.

Reply:
Will add back in then.